common/JackControlAPI.cpp

```diff
@@ -712,6 +801,16 @@ jackctl_wait_signals(jackctl_sigmask_t * sigmask)
         // bugs that cause segfaults etc. during shutdown.
         sigprocmask(SIG_UNBLOCK, &sigmask->signals, 0);
     }
+
+#ifdef __linux__
+fail:
+    for(int i = 0; i < JackFDCount; i++) {
```

Comment:
This and a handful of other lines don't conform to the style of the rest of the code. AFAIK there should be a space after `if`, `for`, etc.